scripts/build.mjs

```diff
@@ -70,7 +80,19 @@ class Builder {
     // environment variable `__COMPAT_LAYER=RunAsInvoker` to force the installer
     // to run as the existing user.
     const env = { ...process.env, __COMPAT_LAYER: 'RunAsInvoker' };
+    const releaseBuildIndex = args.indexOf('--release');
+    const fullBuildVersion = this.getDescribedTag();
+    let ptn;
+
+    if (releaseBuildIndex !== -1) {
+      args.splice(releaseBuildIndex, 1);
+      ptn = /^v?(.*?)-\d+-\w+$/;
```

Comment:
If the commit is a tag, this will fail to match (because the `git describe` output would just be a tag, and not match the `-\d+-\w+` part).

Reply:
I thought that was ok, but if the commit info is missing we end up keeping the leading 'v'. Making the remainder optional fixes that.